ee/codegen/src/__test__/nodes/__snapshots__/conditional-node.test.ts.snap

```diff
@@ -22,6 +22,10 @@ class ConditionalNodeDisplay(BaseConditionalNodeDisplay[ConditionalNode]):
             rhs=None,
         )
     ]
+    condition_ids = {
+        0: UUID("8d0d8b56-6c17-4684-9f16-45dd6ce23060"),
+        1: UUID("ea63ccd5-3fe3-4371-ba3c-6d3ec7ca2b60"),
+    }
```

Comment:
nit: If the keys are always gonna be consecutive integer values starting from 0, we might as well just use an array, no?

Reply:
Good point will update